SHARE/make_perlmutter.inc

```diff
@@ -0,0 +1,163 @@
+#######################################################################
+#            Define Basic Utilities
+#######################################################################
+  SHELL = /bin/sh
+  PWD1 = `pwd`
+  MYHOME = /global/cfs/cdirs/m4505/sw/STELLOPT/bin
+  PRECOMP:= /lib/cpp -traditional -DLINUX
+  COMPILE = $(FC)
+  COMPILE_FREE = $(FC)
+  LINK    = $(FC) -Bstatic -o
+  LINK_AR = ar -ruv
+  LINK_C  = $(CC) -shared -Wl,-z-defs
+
+#######################################################################
+#            Define Compiler Flags
+#######################################################################
+  FLAGS_R = 
+  FLAGS_D = -g -O0 -fexternal-blas -fcheck=all -fbacktrace -Wextra \
+   -Wtarget-lifetime -fbounds-check -ffpe-trap=zero -finit-real=snan \
+   -ffree-line-length-0 -dynamic -fallow-argument-mismatch 
+  LIBS    = 
+  FLAGS_R += -g -fbacktrace -dynamic -fallow-argument-mismatch -ffree-line-length-0
+
+#######################################################################
+#            MPI Options
+#######################################################################
+  LMPI    = T
+  MPI_COMPILE = $(MPIFC)
+  MPI_COMPILE_FREE = $(MPIFC)
+  MPI_COMPILE_C = $(MPICC)
+  MPI_LINK = $(FC)
+  MPI_RUN = srun
+  MPI_RUN_OPTS = -n 68 -c 4 --cpu_bind=cores
+  MPI_RUN_OPTS_SM = -n 68 -c 4 --cpu_bind=cores
+#  MPI_RUN_OPTS_MD = -n 544 -c 4 --cpu_bind=cores
+  MPI_RUN_OPTS_LG = -n 4352 -c 4 --cpu_bind=cores
+
+#######################################################################
+#            NAG Options
+#######################################################################
+  LNAG = F
+  NAG_LIB =
+
+#######################################################################
+#            NETCDF Options
+#######################################################################
+  LNETCDF = T
+  NETCDF_INC = 
+  NETCDF_LIB = 
```

Comment:
You don't need to explicitly set the link or compile include lines?

Reply:
I think it is better to set NETCDF flags here. I will fix.